container-images/cuda/Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:12.6.1-devel-ubi9
```

Comment:
podman doesn't seem to like the shortened version, I think this was tested on docker initially